Integrations/SecurityAdvisor/SecurityAdvisor_description.md

```diff
@@ -0,0 +1,95 @@
+## Overview
```

Comment:
This file should be name `README.md` 
`SecurityAdvisor_description.md` is detailed description about the integration and should only contain info on how to configure the integration instance and where do I get the API KEY.

```
To get your  __API KEY__ log into www.securityadvisor.io and get your key from "My Profile" section or contact us at support@securityadvisor.io
```

Reply:
This is updated

Reply:
Sorry ,probably I didnt explained myself well.
YOu should have 2 files.
1. `README.md` which contains the full integration documentation - follow https://github.com/demisto/content/blob/master/docs/integration_documentation/README.MD
2. SecurityAdvisor_description.md - should contain only info on how to configure the integration instance and where do I get the API KEY.

Reply:
So this file should be named README.md and the other one SecurityAdvisor_description.md

Reply:
done